addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2739,6 +2739,194 @@ options.registry.ReplaceImage = options.Class.extend({
     },
 });
 
+options.registry.ConditionalVisibility = options.Class.extend({
+
+    /**
+     * We clear out the current style tag to make everything visible
```

Comment:
Try to add a docstring when you can on this PR and it should start with a congugated verb to complete the sentence "This function ...". So here "Clears out the...".